design-system/skyyrose-storefront/src/components/Button/button.css

```
@@ -0,0 +1,20 @@
+  align-items: center;
+  justify-content: center;
+  font-family: var(--skyyrose-font-ui, 'Bebas Neue', sans-serif);
+  letter-spacing: var(--tracking-wider, 0.1em);
+  text-transform: uppercase;
+  text-decoration: none;
+  cursor: pointer;
+  border-radius: var(--radius-sm, 4px);
+  transition: var(--transition-smooth, all 0.3s ease);
+}
+.sr-btn[aria-disabled='true'],
+.sr-btn:disabled { opacity: 0.55; cursor: not-allowed; }
```

Stylelint (17.13.0)

[error] Too many declarations, maximum 1 (declaration-block-single-line-max-declarations)

(declaration-block-single-line-max-declarations)